### PR TITLE
DOCS shift to rst - Openvino Ecosystem article update

### PR DESCRIPTION
**Pull request by msmykx-intel**
Fixing directives for code snippets and updating CVAT link.

### CODE REVIEW

File: docs/Documentation/openvino_ecosystem.md [hunk 1]
```diff
@@ -12,70 +12,94 @@
    openvino_docs_tuning_utilities
    workbench_docs_Workbench_DG_Introduction
```
Comment (tsavina) [inline docs/Documentation/openvino_ecosystem.md:13]: can we remove workbench_docs_Workbench_DG_Introduction from toctree for master branch?

File: docs/Documentation/openvino_ecosystem.md [hunk 2]
```diff
@@ -12,70 +12,94 @@
...
-* [Documentation](https://docs.openvino.ai/2022.3/workbench_docs_Workbench_DG_Introduction.html)
-* [Docker Hub](https://hub.docker.com/r/openvino/workbench)
-* [PyPI](https://pypi.org/project/openvino-workbench/)
 
-### OpenVINO™ Training Extensions (OTX)
+* :doc:`Documentation <workbench_docs_Workbench_DG_Introduction>`  
+* `Docker Hub <https://hub.docker.com/r/openvino/workbench>`__  
+* `PyPI <https://pypi.org/project/openvino-workbench/>`__  
+
+OpenVINO™ Training Extensions (OTX)
```
Comment (tsavina) [inline docs/Documentation/openvino_ecosystem.md:72]:
```suggestion
OpenVINO™ Training Extensions 
```

Comment (tsavina) [reply docs/Documentation/openvino_ecosystem.md:72]: according to marketing

File: docs/Documentation/openvino_ecosystem.md (hunk 1 above)
Comment (msmykx-intel) [inline docs/Documentation/openvino_ecosystem.md:13]:
```suggestion
```

Comment (msmykx-intel) [reply docs/Documentation/openvino_ecosystem.md]: Done in https://github.com/openvinotoolkit/openvino/pull/16050/commits/343622d822df5737c086006d0a9dcbb0a2933221

**Review by sgolebiewski-intel — APPROVED** [review]

**Pull request merged**